src/components/SecondaryPanes/CommandBar.css

```diff
@@ -49,7 +49,7 @@ img.skipPausing {
 }
 
 .command-bar img.stepOut {
-  mask: url(/images/stepOut.svg) no-repeat;
+  mask: url(/mc/command-chevron.svg) no-repeat;
```

Comment:
this was sample code for the svg... we'll need to add the chevron to the correct button in the tabs bar :)